### ISSUE TITLE
Command maint:status fails displaying help

### ISSUE THREAD

**Issue by andypost** [drush-ops/drush#6117]
**Describe the bug**
Running `drush maint:status --help` fails

**To Reproduce**
display help of `maint:status` command

**Expected behavior**
should work

**Actual behavior**
````
$ drush maint:status --help
 [warning] DOMDocument::createElement(): unterminated entity reference    & drush cron XmlDescriptor.php:69
 [warning] DOMDocument::createElement(): unterminated entity reference    & drush cron HelpDocumentBuilder.php:36
Fail if maintenance mode is enabled.

This commands fails with exit code of 3 when maintenance mode is on. This special
 exit code distinguishes from a failure to complete.

Examples:
 [warning] Undefined variable $eample HelpCLIFormatter.php:41
[ERROR] [php] [2024-09-25T13:08:52] TypeError: Cannot access offset of type string on string in Drush\Commands\help\HelpCLIFormatter->write() (line 42 of /var/www/html/vendor/drush/drush/src/Commands/help/HelpCLIFormatter.php) #0 /var/www/html/vendor/consolidation/output-formatters/src/FormatterManager.php(242): Drush\Commands\help\HelpCLIFormatter->write()

```

**Workaround**
Is there another way to do the desired action?

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 13.x/12.x/11.x/10.x/8.x (please be specific, and try latest release)
| Drupal version? | 11.x/10.x/9.x/8.x/7.x
| PHP version     | 8.x/7.x
| OS?             | Mac/Linux/Windows

**Additional information**
Add any other context about the problem here.

**Comment by andypost** [comment]
Feedback from Barcelona's BOF

**Comment by andypost** [comment]
instead of array it getting `$example` as string `string(53) "Only run cron when Drupal is not in maintenance mode."`

the whole array for Druah 13.2
```
array(8) {
  ["id"]=>
  string(12) "maint:status"
  ["name"]=>
  string(12) "maint:status"
  ["usages"]=>
  array(1) {
    [0]=>
    string(57) "maint:status [--notify] [--xh-link XH-LINK] [--druplicon]"
  }
  ["examples"]=>
  array(1) {
    [0]=>
    string(53) "Only run cron when Drupal is not in maintenance mode."
  }
  ["description"]=>
  string(36) "Fail if maintenance mode is enabled."
  ["options"]=>
  array(3) {
    ["notify"]=>
    array(6) {
      ["name"]=>
      string(8) "--notify"
      ["shortcut"]=>
      string(0) ""
      ["accept_value"]=>
      string(1) "0"
      ["is_value_required"]=>
      string(1) "0"
      ["is_multiple"]=>
      string(1) "0"
      ["description"]=>
      string(103) "Notify upon command completion. If set to a number, commands that finish in fewer seconds won't notify."
    }
    ["xh-link"]=>
    array(6) {
      ["name"]=>
      string(9) "--xh-link"
      ["shortcut"]=>
      string(0) ""
      ["accept_value"]=>
      string(1) "1"
      ["is_value_required"]=>
      string(1) "1"
      ["is_multiple"]=>
      string(1) "0"
      ["description"]=>
      string(31) "URL to your XHProf report site."
    }
    ["druplicon"]=>
    array(6) {
      ["name"]=>
      string(11) "--druplicon"
      ["shortcut"]=>
      string(0) ""
      ["accept_value"]=>
      string(1) "0"
      ["is_value_required"]=>
      string(1) "0"
      ["is_multiple"]=>
      string(1) "0"
      ["description"]=>
      string(42) "Shows the druplicon as glorious ASCII art."
    }
  }
  ["help"]=>
  string(134) "This commands fails with exit code of 3 when maintenance mode is on. This special
 exit code distinguishes from a failure to complete."
  ["aliases"]=>
  array(1) {
    [0]=>
    string(7) "mstatus"
  }
}
```

**Comment by weitzman** [comment]
Fixed in #6118

**Comment by andypost** [comment]
Thank you 👍
